packages/cli/src/utils/codegen.ts

```diff
@@ -0,0 +1,10 @@
+import path from "path";
+import { getForgeConfig } from "./foundry.js";
+
+/**
+ * The directory that all MUD auto-generated files are placed in.
+ */
+export async function getCodegenDirectory(profile?: string) {
+  const srcDirectory = (await getForgeConfig(profile)).src;
+  return path.join(srcDirectory, "codegen");
+}
```

Comment:
this might be better of as part of store config, rather than a hardcoded standalone utility